string                    waypoint_id
int64                     instance_number
---
string[]                  index_to_label_name
int32[]                   label
float32[]                 label_probabilities
float32[]                 label_frequencies
geometry_msgs/Point32[]   points
